TemplateDev/settings.py

```diff
@@ -81,6 +80,17 @@
     }
 }
 
+# DATABASES = {
```

Comment:
Ye database waala part, user/password aise nhi daalte. Like this is an environment variable thing. 

Agle commits mai hata dena, this is different for everyone. So they shall set it locally when it is being run in their systems.